pkg/agent/proxy/proxier.go

```diff
@@ -639,6 +643,16 @@ func (p *metaProxierWrapper) GetServiceFlowKeys(serviceName, namespace string) (
 	return append(v4Flows, v6Flows...), append(v4Groups, v6Groups...), v4Found || v6Found
 }
 
+func (p *metaProxierWrapper) GetServiceByIP(serviceStr string) (k8sproxy.ServicePortName, bool) {
+	// Format of serviceStr is fmt.Sprintf("%s:%d/%s", clusterIP, svcPort, protocol).
```

Comment:
that's a weird way of presenting it, why not:

```
// Format of serviceStr is <clusterIP>:<svcPort>/<protocol>
```

Reply:
Thanks, addressed.